text/src/autogluon/text/automm/optimization/utils.py

```diff
@@ -211,11 +211,34 @@ def get_weight_decay_param_names(model: nn.Module):
     -------
     A list of parameter names not using weight decay.
     """
-    decay_param_names = get_parameter_names(model, [nn.LayerNorm])
+    # By default, we should not apply weight decay for all the norm layers
+    decay_param_names = get_parameter_names(model,
+                                            [nn.LayerNorm, nn.BatchNorm1d, nn.BatchNorm2d, nn.BatchNorm3d,
```

Comment:
Besides layer norm, do we have references about not using weight decay on other norms?

Reply:
In https://arxiv.org/pdf/1812.01187.pdf (Section "No bias decay"), the author found that do not add wd to \gamma and \beta in BN layers will improve the performance.